src/pages/admin/postings/edit.vue

```diff
@@ -56,6 +56,10 @@ if (isUpdating && posting) {
 const isSubmitting = ref(false);
 
 const onSubmit = handleSubmit(async values => {
+
+  if (!isUpdating) {
+    return;
+  }
```

Comment:
delete this change.

Reply:
@amandesai01 Please check I have removed the change